plugins/optimization-detective/tests/test-detection.php

```diff
@@ -225,4 +225,27 @@ public function test_od_get_detection_script_returns_script( Closure $set_up, ar
 			$this->assertStringNotContainsString( '"restApiNonce":', $script );
 		}
 	}
+
+	/**
+	 * Test od_create_rest_url_metric_store_rest_request().
+	 *
+	 * @covers ::od_create_rest_url_metric_store_rest_request
+	 */
+	public function test_od_create_rest_url_metric_store_rest_request(): void {
+
+		global $wp_rest_server;
+
+		if ( ! isset( $wp_rest_server ) ) {
+			$wp_rest_server = new WP_REST_Server();
+
+			// Run the action to register the routes.
+			do_action( 'rest_api_init' );
+		}
+
+		// Get the routes.
+		$routes = $wp_rest_server->get_routes();
```

Comment:
This should rather do the following:

```suggestion
		$routes = rest_get_server()->get_routes();
```

If WordPress doesn't clean up the `$wp_rest_server` global then we should do it in `set_up`/`tear_down`, for example, do `unset( $GLOBALS['wp_rest_server'] );`.